scan/src/components/PieChart.re

```diff
@@ -0,0 +1,63 @@
+module Styles = {
+  open Css;
+
+  let pie = (size, color) =>
+    style([
+      overflow(`hidden),
+      display(`flex),
+      position(`relative),
+      width(`px(size)),
+      height(`px(size)),
+      borderRadius(`percent(100.)),
+      backgroundColor(color),
+    ]);
+
+  let segment = (offset, asize, color, isHidden) =>
+    style([
+      visibility(isHidden ? `hidden : `visible),
+      overflow(`hidden),
+      width(`percent(100.)),
+      height(`percent(100.)),
+      position(`absolute),
+      transforms([
+        `translate((`zero, `percent(-50.))),
+        `rotate(`deg(90.)),
+        `rotate(`deg(offset)),
```

Comment:
duplicated?

Reply:
Transformation can be combine. We just rotate 90(deg) and then rotate offset(deg).